provisioning/roles/kuma/files/apache2/all-servers.conf

```diff
@@ -3,15 +3,6 @@
 # Our servers symlink to this file. Do not move it without first coordinating
 # with WebOps.
 
-<Location /static/fonts>
-    # Only enable CORS for fonts when the request is coming from an approved domain
-    Header unset Access-Control-Allow-Origin
-    SetEnvIf Origin "https?://(.*\.(mozilla|allizom)\.(com|org|net))" CORS=$0
-    SetEnvIf Origin "https?://(mdn\.mozillademos\.org)" CORS=$0
-    SetEnvIf Origin "https?://(kuma\.readthedocs\.org)" CORS=$0
-    Header set Access-Control-Allow-Origin %{CORS}e env=CORS
```

Comment:
Note, Whitnoise will add `Access-Control-Allow-Origin=*` to all files it serves by default. If you think we should disable it, you can use the `WHITENOISE_ALLOW_ALL_ORIGINS` setting (http://whitenoise.readthedocs.org/en/latest/django.html?highlight=WHITENOISE_ALLOW_ALL_ORIGINS#WHITENOISE_ALLOW_ALL_ORIGINS) or pass `allow_all_origins=False` to the instantiation of `KumaDjangoWhiteNoise` in kuma.wsgi.